### PR TITLE
document workaround for conformance tests on Mac

### PR DESCRIPTION
**Pull request by arkodg**
Relates to https://github.com/envoyproxy/gateway/issues/284

Signed-off-by: Arko Dasgupta <arko@tetrate.io>

### CODE REVIEW

**Comment by arkodg** [comment]
https://github.com/envoyproxy/gateway/issues/323 will be needed to replace the local manifest to point to a custom image rather than `envoyproxy/gateway-dev:latest`

**Review by AliceProxy — APPROVED** [review]
lgtm. I've been doing using Kubernetes clusters and doing this for development so should be a fine workaround.

File: DEVELOPER.md [hunk 1]
```diff
@@ -59,8 +59,11 @@ with the `envoyproxy/gateway-dev:latest` Envoy Gateway image into a Kubernetes c
 local Kind cluster. Go [here](https://gateway-api.sigs.k8s.io/concepts/conformance/) to learn
 more about the tests.
 
-**_NOTE:_**  This command is currently Work in Progress. :construction::construction::construction::construction:
-
+**_NOTE:_** Conformance tests against a kind cluster is currently unsupported on Mac computers.
+As a workaround, you could run this against your own Kubernetes cluster (such as Kubernetes on Docker Desktop) using this command -
+`IMAGE=docker.io/you/gateway-dev make push-multiarch && IMAGE=docker.io/you/gateway-dev make kube-deploy && make run-conformance`
+which builds and pushes the Envoy-Gateway image to your hub, deploys Envoy Gateway resources into your cluster
+and run the Gateway API conformance tests.
```
Comment (danehans) [inline DEVELOPER.md:66]: s/run/runs/

File: DEVELOPER.md [hunk 2]
```diff
@@ -72,5 +75,6 @@ Now you are able to view the running Envoy configuration by navigating to `127.0
 
 There are many other endpoints on the [Envoy admin interface](https://www.envoyproxy.io/docs/envoy/v1.23.0/operations/admin#operations-admin-interface) that may be helpful when debugging.
 
+
```
Comment (danehans) [inline DEVELOPER.md:78]: Unneeded extra line added. Please remove.

**Review by danehans — CHANGES_REQUESTED** [review]

**Review by danehans — APPROVED** [review]

**Pull request merged**